L1Trigger/TrackFindingTMTT/interface/L1fittedTrack.h

```diff
@@ -333,27 +333,19 @@ namespace tmtt {
 
     // Is the fitted track trajectory within the same (eta,phi) sector of the HT used to find it?
     bool consistentSector() const {
-      if (settings_->hybrid()) { 
+      if (settings_->hybrid()) {
         float phiCentre = 2. * M_PI * iPhiSec() / settings_->numPhiSectors();
         float sectorHalfWidth = M_PI / settings_->numPhiSectors();
         bool insidePhi =
             (reco::reduceRange(std::abs(reco::deltaPhi(this->phiAtChosenR(done_bcon_), phiCentre))) < sectorHalfWidth);
         return insidePhi;
       } else {
-        bool insidePhi = 
-            (std::abs(reco::deltaPhi(this->phiAtChosenR(done_bcon_), secTmp_->phiCentre())) < secTmp_->sectorHalfWidth());
-        bool insideEta = 
+        bool insidePhi = (std::abs(reco::deltaPhi(this->phiAtChosenR(done_bcon_), secTmp_->phiCentre())) <
+                          secTmp_->sectorHalfWidth());
+        bool insideEta =
             (this->zAtChosenR() > secTmp_->zAtChosenR_Min() && this->zAtChosenR() < secTmp_->zAtChosenR_Max());
         return (insidePhi && insideEta);
       }
-    } 
-
-
-    bool hybridConsistentSector() const {
-      std::cout << "The sector of this track is " << iPhiSec() << std::endl;
-      bool insidePhi =
-          (std::abs(reco::deltaPhi(this->phiAtChosenR(done_bcon_), 2. * M_PI * iPhiSec() / float(settings_->numPhiSectors()))) < M_PI / settings_->numPhiSectors());
-      return (insidePhi);
```

Comment:
I don't quite follow why this code was edited - can you explain it to me?

Reply:
Yes, ```secTmp_``` used in this function is only defined when not using hybrid, so I made it so that when it is hybrid it uses the definition of the functions that are called from ```secTmp_```.

Reply:
OK, maybe @tomalin can comment -- is this what you were expecting, or did you want Daniel to make a new algorithm in TrackFindingTracklet?

Reply:
Looks OK as is.